MultimodalQnA/tests/test_compose_on_gaudi.sh

```diff
@@ -63,7 +64,8 @@ function start_services() {
 
 function prepare_data() {
     cd $LOG_PATH
-    echo "Downloading video"
+    echo "Downloading image and video"
+    wget https://github.com/docarray/docarray/blob/main/tests/toydata/image-data/apple.png?raw=true -O ${image_fn}
```

Comment:
It seems like we should be deleting this (and the video file) after we are done the file